services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -902,6 +932,12 @@ public MapboxDirections build() {
         approaches(formattedApproaches);
       }
 
+      if (walkingOptions() == null) {
+        walkingOptions(WalkingOptions.builder().build());
```

Comment:
Is `WalkingOptions` `@Nullable`? Do we actually need to pass in a non-null object then?

Reply:
@Guardiola31337 this is so I don't have to do a null check [here](https://github.com/mapbox/mapbox-java/pull/991/files/6ab38103da15a761fcccc0acb36c9b88be878841#diff-7ee26788285a9a937ff59dfdcaff423aR111)